Makefile.in

```diff
@@ -44,6 +44,7 @@ RKT_LOCAL_COREOS_PXE_IMAGE_PATH := @RKT_LOCAL_COREOS_PXE_IMAGE_PATH@
 RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER := @RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER@
 
 RKT_STAGE1_COREOS_BOARD := @RKT_STAGE1_COREOS_BOARD@
+RKT_STAGE1_COREOS_VERSION := @RKT_STAGE1_COREOS_VERSION@
```

Comment:
I'm not sure I'm happy with this change:

1. It isn't really a general value, its use is contained entirely inside coreos-common.mk.
2. Now, if we want to upgrade the version of coreos image, we have to do the modifications in both configure.ac and coreos-common.mk.
3. Might be misleading, because it will be ignored if we use the `--with-coreos-local-pxe-image-*` flags.